recipes/ensmallen/all/conanfile.py

```diff
@@ -25,7 +25,7 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("armadillo/12.2.0")
+        self.requires("armadillo/12.6.4")
```

Comment:
this is ok as the only other using armadillo in CCI is gdal and already has the `12.6.4` [as dependency](https://github.com/conan-io/conan-center-index/blob/1634381974b91b14804ce996c00fdd1374a29b72/recipes/gdal/post_3.5.0/conanfile.py#L190).